src/App.jsx

```diff
@@ -14,70 +15,80 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, chahgeSelectedGood] = useState('Jam');
+  const [title, setTitle] = useState(`${selectedGood} is selected`);
+  const changeGood = (good, text) => {
+    chahgeSelectedGood(good);
+    setTitle(text);
+  };
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
+  return (
+    <main className="section container">
+      <h1 className="title is-flex is-align-items-center">
+        {title}
+        {title !== 'No goods selected' && (
+          <button
+            data-cy="ClearButton"
+            type="button"
+            className="delete ml-3"
+            onClick={() => changeGood('', 'No goods selected')}
```

Comment:
```suggestion
            onClick={() => changeGood('', RESET)}
```
Consider using variables instead of magic values